pyaerocom/aeroval/modelmaps_engine.py

```diff
@@ -85,6 +88,9 @@ def _run_model(self, model_name: str, var_list):
                 _files = self._process_map_var(model_name, var, self.reanalyse_existing)
                 files.extend(_files)
 
+            except TypeError as ex:
+                logger.warning(f"{ex}")
+                return files
```

Comment:
The warning could be a bit more explicit, i.e. include {model_name} and {var}, e.g. `f"Exception during processing of  model for {model_name} ({var}): {ex}"` (You can't be sure that the logger.info in line 85 is written, depending on logging configuration.)

I think `return files` is wrong here. Not sure why you get the TypeError, but I guess you want to extend the other `var` in `var_list`. The real return is in line 103.

Reply:
true, I want a continue, in reality it does not matter tho....  
I'll chat you in person